odoo/service/db.py

```diff
@@ -114,11 +114,26 @@ def _create_empty_database(name):
                 sql.Identifier(name), collate, sql.Identifier(chosen_template)
             ))
 
+    # TODO: add --extension=trigram,unaccent
+    try:
+        db = odoo.sql_db.db_connect(name)
+        with closing(db.cursor()) as cr:
+            cr.execute("CREATE EXTENSION IF NOT EXISTS pg_trgm")
+            cr.commit()
+    except psycopg2.Error:
+        pass
+
     if odoo.tools.config['unaccent']:
         try:
             db = odoo.sql_db.db_connect(name)
             with closing(db.cursor()) as cr:
                 cr.execute("CREATE EXTENSION IF NOT EXISTS unaccent")
+                # In the point of view of PostgreSQL, It is not correct because
+                # `unaccent` depend of external data - see
```

Comment:
```suggestion
                # From PostgreSQL's point of view, making 'unaccent' immutable is not correct
                # because it depends on external data - see
```